### ISSUE TITLE
WebSQL Todo list tutorial errors

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#495]
```
Please describe the issue:
http://www.html5rocks.com/tutorials/webdatabase/todo/

There are a few errors in the tutorial code that do NOT appear in the 
demonstration code embedded in this page. There is an extra single quote on the 
second to last line of Step 4a.
And the 3rd parameter of tx.executeSql() in Step 5 should be 
html5rocks.webdb.onSuccess, NOT loadTodoItems.

Please provide any additional information below.

```

Original issue reported on code.google.com by `alphapol...@gmail.com` on 26 Mar 2011 at 4:24

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 26 Mar 2011 at 6:43
- Added labels: **Milestone-5**, **tu**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 18 Apr 2011 at 4:55
- Added labels: **Milestone-6**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 8 Jun 2011 at 1:04
- Added labels: **Tutorial**
- Removed labels: **tu**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Revision 5ace28a92327.
```

Original comment by `han...@google.com` on 19 Sep 2011 at 11:35
- Changed state: **Fixed**
- Added labels: **Milestone-7.2**
- Removed labels: **Milestone-6**
